src/routes/Rockets.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { cancelReserveRocket, reserveRocket, getRockets } from '../components/redux/rockets/rocketsSlice';
+import styles from '../styles/Rockets.module.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const { rockets, isLoading, error } = useSelector((state) => state.rockets);
+  useEffect(() => {
+    if (rockets.length === 0) {
+      dispatch(getRockets());
+    }
+  }, [dispatch, rockets.length]);
+
+  const handleReserveRocket = (id) => {
+    dispatch(reserveRocket(id));
+  };
+  const handleCancelReserveRocket = (id) => {
+    dispatch(cancelReserveRocket(id));
+  };
+  if (error) return <h2>Something went wrong</h2>;
+
+  return isLoading ? (
+    <h1>Loading...</h1>
+  ) : (
+    <>
+      <hr />
+      <div className={styles.rocketsWrapper}>
+        {rockets.map((rocket) => (
+          <div key={rocket.rocket_id} className={styles.rocket}>
+            <img className={styles.image} src={rocket.flickr_images[0]} alt="" />
+            <div className={styles.rocketInfo}>
+              <h2>{rocket.rocket_name}</h2>
+              <p className={styles.rocketDescription}>
+                {rocket.reserved ? (
+                  <span className={styles.reserved}>
+                    <Badge bg="success">Reserved</Badge>
+                  </span>
+                ) : null}
+
+                {rocket.description}
+              </p>
+              {rocket.reserved ? (
+                <Button
+                  variant="outline-secondary"
+                  onClick={() => handleCancelReserveRocket(rocket.rocket_id)}
+                >
+                  Cancel Reservation
+                </Button>
+              ) : (
+                <Button
+                  variant="primary"
+                  onClick={() => handleReserveRocket(rocket.rocket_id)}
+                >
+                  Reserve Rocket
+                </Button>
+              )}
+
+            </div>
+          </div>
+        ))}
+      </div>
+
+    </>
```

Comment:
- [ ] @Bifabig and @Mar12358 great work on your rockets page, however, the color of your nav items, reserved icon and cancel reservation button don't match with the design given check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/a2175dee-3d29-4fee-8d14-42888879234c) | ![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/4e6dc09e-51fe-4d7a-ae97-ac87e6e742d7)

Reply:
@danmainah kindly take note that the cancel button is exactly like the design when you're not hovering over it. It changes color on hover that's the one you have in your screenshot